src/main/cpp/archive_utils.h

```diff
@@ -29,26 +30,12 @@ void DetermineArchiveContents(const std::string &archive_path,
                               std::vector<std::string> *files,
                               std::string *install_md5);
 
-struct DurationMillis {
- public:
-  const uint64_t millis;
-
-  DurationMillis() : millis(kUnknownDuration) {}
-  DurationMillis(const uint64_t ms) : millis(ms) {}
-
-  bool IsUnknown() const { return millis == kUnknownDuration; }
-
- private:
-  // Value representing that a timing event never occurred or is unknown.
-  static constexpr uint64_t kUnknownDuration = 0;
-};
-
 // DurationMillis that tracks if an archive was extracted.
 struct ExtractionDurationMillis : DurationMillis {
```

Comment:
I wonder if we really need this class: we either did an extraction and know the time it took, or we didn't and we don't; we never inspect `archive_extracted`, except in tests. Wouldn't a `DurationMillis` suffice, with "unknown" signifying "not extracted"?